data/azure/regions.json

```diff
@@ -0,0 +1,335 @@
+[
+  {
+    "name": "East US",
+    "region": "eastus",
+    "cheaper": "eastus2"
+  },
+  {
+    "name": "East US 2",
+    "region": "eastus2"
```

Comment:
For the same of consistency, we should include every field in every entry.

For fields that are not relevant, because they don't exist, they should have a null value.

Example:

{
    "name": "South Central US",
    "region": "southcentralus",
    "cheaper": null
  }